src/opnsense/mvc/app/controllers/OPNsense/Syslog/LogviewController.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/**
+ *    Copyright (C) 2016 Deciso B.V.
+ *
+ *    All rights reserved.
+ *
+ *    Redistribution and use in source and binary forms, with or without
+ *    modification, are permitted provided that the following conditions are met:
+ *
+ *    1. Redistributions of source code must retain the above copyright notice,
+ *       this list of conditions and the following disclaimer.
+ *
+ *    2. Redistributions in binary form must reproduce the above copyright
+ *       notice, this list of conditions and the following disclaimer in the
+ *       documentation and/or other materials provided with the distribution.
+ *
+ *    THIS SOFTWARE IS PROVIDED ``AS IS'' AND ANY EXPRESS OR IMPLIED WARRANTIES,
+ *    INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY
+ *    AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+ *    AUTHOR BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY,
+ *    OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ *    SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ *    INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ *    CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ *    ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ *    POSSIBILITY OF SUCH DAMAGE.
+ *
+ */
+
+namespace OPNsense\Syslog;
+
+use \OPNsense\Syslog\Syslog;
+
+
+/**
+ * Class LogViewController
+ * @package OPNsense\Syslog
+ */
+class LogviewController extends \OPNsense\Base\IndexController
+{
+
+    /**
+     * Syslog Log dummy view page
+     */
+    public function indexAction()
+    {
+        $this->view->title = gettext('Log File: none');
+        $this->view->pick('OPNsense/Syslog/logview');
+    }
+
+    /**
+     * Custom Log view page
+     */
+    public function showlogAction($name, $filter = '')
+    {
+        $this->view->title = gettext('Log File: ' . $name);
```

Comment:
this is very bad as it will never go into the translation file.
Use `sprintf(gettext('Log File: %s'), $name)` instead